Source/Eto.Wpf/Forms/Controls/SplitterHandler.cs

```diff
@@ -252,14 +250,18 @@ public int Position
 				}
 				if (splitter.ResizeDirection == swc.GridResizeDirection.Columns)
 				{
-					var controlWidth = Control.IsLoaded ? Control.ActualWidth : Control.Width;
+					var col1 = Control.ColumnDefinitions[ 0 ];
```

Comment:
Spaces inside [ brackets ] are not default formatting - happened during the move of those two lines to inner block.